research/berry/berrylib/test_berry.py

```diff
@@ -182,7 +187,8 @@ def test_fast_inla(method, N=10, iterations=1):
         start = time.time()
         out = inla_model.inference(y_i, n_i, method=method)
         end = time.time()
-        np.testing.assert_allclose(out[0][0].sum(), 2111.808, rtol=1e-3)
```

Comment:
Wait, this line was failing before and you just deleted it? Was this just a wrong check?